packages/wrangler/src/utils/print-bindings.ts

```diff
@@ -8,31 +9,31 @@ export const friendlyBindingNames: Record<
 	keyof CfWorkerInit["bindings"],
 	string
 > = {
-	data_blobs: "Data Blobs",
-	durable_objects: "Durable Objects",
-	kv_namespaces: "KV Namespaces",
+	data_blobs: "Data Blob",
+	durable_objects: "Durable Object",
+	kv_namespaces: "KV Namespace",
 	send_email: "Send Email",
-	queues: "Queues",
-	d1_databases: "D1 Databases",
-	vectorize: "Vectorize Indexes",
-	hyperdrive: "Hyperdrive Configs",
-	r2_buckets: "R2 Buckets",
+	queues: "Queue",
+	d1_databases: "D1 Database",
+	vectorize: "Vectorize Index",
+	hyperdrive: "Hyperdrive Config",
+	r2_buckets: "R2 Bucket",
 	logfwdr: "logfwdr",
-	services: "Services",
-	analytics_engine_datasets: "Analytics Engine Datasets",
-	text_blobs: "Text Blobs",
+	services: "Worker",
+	analytics_engine_datasets: "Analytics Engine Dataset",
+	text_blobs: "Text Blob",
 	browser: "Browser",
 	ai: "AI",
 	images: "Images",
 	version_metadata: "Worker Version Metadata",
 	unsafe: "Unsafe Metadata",
-	vars: "Vars",
-	wasm_modules: "Wasm Modules",
-	dispatch_namespaces: "Dispatch Namespaces",
-	mtls_certificates: "mTLS Certificates",
-	workflows: "Workflows",
-	pipelines: "Pipelines",
-	secrets_store_secrets: "Secrets Store Secrets",
+	vars: "Environment Variable",
```

Comment:
Hmm not sure about this wording? Why not just `Variable`?
Environment variable suggests `process.env` to me.

Reply:
I was going off what we call them in the docs: https://developers.cloudflare.com/workers/configuration/environment-variables/

Reply:
🤢  - okey dokey!